From: Jens Axboe <jens.axboe@oracle.com>
Date: Fri, 4 Dec 2009 10:07:38 +0100
Subject: [PATCH] io controller: quick fix for blk-cgroup and modular CFQ
Git-commit: 237e5bc4e51813e9d8ba9da0f63e7acc608882d7
References: FATE#311054
Patch-Mainline: 2.6.33

It's currently not an allowed configuration, so express that in Kconfig.

Signed-off-by: Jens Axboe <jens.axboe@oracle.com>
Signed-off-by: Hannes Reinecke <hare@suse.de>
---
 block/Kconfig.iosched |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/block/Kconfig.iosched b/block/Kconfig.iosched
index b71abfb..5368d74 100644
--- a/block/Kconfig.iosched
+++ b/block/Kconfig.iosched
@@ -34,7 +34,7 @@ config IOSCHED_CFQ
 
 config CFQ_GROUP_IOSCHED
 	bool "CFQ Group Scheduling support"
-	depends on IOSCHED_CFQ && CGROUPS
+	depends on IOSCHED_CFQ=y && CGROUPS
 	select BLK_CGROUP
 	default n
 	---help---
-- 
1.6.0.2

